src/Apps/Artwork/Components/SaveButton.tsx

```diff
@@ -0,0 +1,24 @@
+import { color, Flex } from "@artsy/palette"
+import Icon from "Components/Icon"
+import React from "react"
+import styled from "styled-components"
+
+const SaveIconWrapper = styled(Flex)`
+  position: relative;
+  justify-content: center;
+`
```

Comment:
While there is value in idiomatic styled components naming, it's a bit simpler to just use flex directly and inline the props: 

```jsx
<Flex justifyContent='center' position='relative'>
  <SaveIcon name="heart" />
</Flex>
```

Reply:
Good call.